lib/adal-aad-node/src/engine/AzureDeviceCode.ts

```diff
@@ -0,0 +1,112 @@
+import { AzureAuth } from "./AzureAuth";
```

Comment:
nit: But you're inconsistent with the casing of your files. Some are capitalized, while some are camelCased

Reply:
Usually for single class files I use capital letters but I can change that.